webpack.config.js

```diff
@@ -185,16 +185,30 @@ var config = {
     '#cheap-module-eval-source-map'
 };
 
+
+function hasZopfli() {
```

Comment:
fwiw we probably need this in getsentry and sentry-plugins, or we need to not use zopfli in those